assets/scss/base.scss

```diff
@@ -238,76 +190,62 @@ a.v-btn {
 .account-container {
   width: 100%;
   flex-wrap: wrap !important;
-}
-.account-container h1 {
-  text-align: center;
-  padding-top: 50px;
-}
-.menu-card {
-  padding-top: 30px;
-  padding-right: 20px;
-  background-color: #f1f3f5 !important;
-}
-.menu-list-text {
-  font-weight: bold;
-  color: $gray7;
-  font-size: 1rem;
-}
-.menu-header {
-  flex: 0 3 auto;
-  color: #fff;
-  background: #38598a;
-  font-weight: bold;
-  font-size: 1rem;
-  padding-left: 30px;
-  padding-top: 12px !important;
-}
-.menu-item {
-  flex: 0 3 auto;
-  font-size: 1rem !important;
-  line-height: 1.5rem;
-  color: $gray7;
-}
-.menu-item-title {
-  flex: 0 3 auto;
-  font-size: 1.125rem !important;
-  font-weight: bold !important;
-}
-.menu-title {
-  font-size: 2rem;
-  font-weight: bold;
-  color: #212529;
-  text-align: center;
-}
-.menu-info {
-  font-size: 1rem;
-  line-height: 1.5rem;
-  color: #495057;
-  text-align: center;
-  max-width: 850px;
-}
-.menu-info-bold {
-  font-size: 1.125rem;
-  font-weight: bold;
-  color: #212529;
-  text-align: center;
-}
-.menu-title-badge {
-  color: #003366 !important;
-  background: $BCgovGold5 !important;
-  font-weight: bold;
-  font-size: 0.875rem;
-  text-decoration: none !important;
-  text-align: center;
-  border-bottom-left-radius: 0.25rem;
-  border-bottom-right-radius: 0.25rem;
-  height: 35px !important;
+  padding: 70px 0;
+
+  .v-card__title {
```

Comment:
I renamed various classes here to avoid duplication: Vuetify already assigns classnames to its components, so I'm using those whenever possible now. This also avoid our own classes from conflicting with Vuetify's.